### PR TITLE
Currency : Added ubi9 script for sentry-python

### PR DESCRIPTION
**Pull request by stutiibm**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: s/sentry-python/sentry-python_ubi_9_3.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+#variables
+PACKAGE_NAME=sentry-python
+PACKAGE_VERSION=${1:-2.7.1}
+PACKAGE_URL=https://github.com/getsentry/sentry-python
+
+sudo dnf install -y ncurses wget git python3 python3-devel make gcc gcc-c++
+
+# miniconda installation
+wget https://repo.anaconda.com/miniconda/Miniconda3-py311_23.10.0-1-Linux-ppc64le.sh -O miniconda.sh
```
Comment (nikhil-kalbande) [inline s/sentry-python/sentry-python_ubi_9_3.sh:32]: @stutiibm  Why we need to use conda environment, I dont see we installing any conda package

File: s/sentry-python/build_info.json [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-    "build_script": "sentry-python_1.12.1_ubi_8.7.sh"
-  },
-  "0.*.*": {
-    "build_script": "sentry-python_ubi_8.4.sh"
-  }
-}
\ No newline at end of file
+  "validate_build_script": true,
+  "use_non_root_user": true,
+  "2.*": {
```
Comment (nikhil-kalbande) [inline s/sentry-python/build_info.json:12]: lets keep old build script supported for old version.. or new script works for 1.12.1 also?

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: s/sentry-python/sentry-python_ubi_9_3.sh (hunk 1 above)
Comment (stutiibm) [reply s/sentry-python/sentry-python_ubi_9_3.sh]: Replaced with python virtual environment.

File: s/sentry-python/build_info.json (hunk 2 above)
Comment (stutiibm) [reply s/sentry-python/build_info.json:12]: Added support for older script

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**